src/twisted/web/test/test_webclient.py

```diff
@@ -1381,3 +1395,101 @@ def test_hostBracketIPv6AddressLiteral(self):
         self.assertEqual(uri.host, b"::1")
         self.assertEqual(uri.netloc, b"[::1]:80")
         self.assertEqual(uri.toBytes(), b'http://[::1]:80/index.html')
+
+
+
+class DeprecationTests(unittest.TestCase):
+    """
+    Tests that l{getPage} and friends are deprecated.
```

Comment:
1. `L{...}`
2. It seems like this should use the namespace `L{client.getPage}` as below.